### ISSUE TITLE
Fix request: false positive in NotProperlySynchronizedOnFieldWhileUsingGuardedBy 

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#442]
There is another rule for checking "this"
```
class FalsePos {
        @GuardedBy("this")
        private String txt;

        public String getText() {
            return txt; // <-- false positive
        }

        public void setText(String t) {
            txt = t; //  <-- false positive
        }
}

```
